### DISCUSSION TITLE
It is possible to automatic sign in the user after api signup ?

### DISCUSSION THREAD

**Discussion by antoniodinovi** [logto-io/logto#3090] [Q&A]
Hi all, i have an application with a custom registration flow. After my flow i register the user using the logto api POST on /api/users. This works fine for me.
After this signup the user is redirect to the sign-in screen and thei he just login in to the app.
There is some way to automatically login the user after the programmatically signup, and skip the sign-in page redirect ? 
my application front end is based on next.js

Thanks in advance

**Comment by gao-sun** [comment]
Logto is OIDC-based, so initiate an OIDC auth is required for end-user sign-in.

for now we have no out-of-the-box support for deep sign-in customization - you need to update the `packages/ui` manually, or replace the whole process with your own.

we've add the support of full sign-in customization (i.e. headless) to our roadmap, but it's at a medium priority.
